modules/cluster-logging-collector-fluentd.adoc

```diff
@@ -7,11 +7,6 @@
 
 You can use the Fluentd *forward* plug-ins to send a copy of your logs to an external log aggregator, instead of the default Elasticsearch.
 
-[NOTE]
-====
-You can configure {product-title} to send logs to both the internal Elasticsearch instance and an outside log aggregator using the '@type copy' plug-in within the Fluentd `output-application` configuration file. Specific instructions for this configuration are beyond the scope of this documentation.
```

Comment:
That is ok to drop these lines. In fact, the 'copy' plugin are still be used in background.